src/api/models/field.js

```diff
@@ -140,6 +140,14 @@ export default async db => {
     collection.removeById = id =>
         collection.remove({ _id: new ObjectID(id), name: { $ne: 'uri' } });
 
+    collection.removeBySubresource = subresourceId =>
+        collection.remove({
+            $or: [
+                { subresourceId: new ObjectID(subresourceId) },
+                { subresourceId },
```

Comment:
I'm surprised that works without an error in mongo, depending on the type of the variable ou are giving.
Are to two cases really used ?

Reply:
Yes, if not it doesn't delete one of those